Add PERMISSIONS.md to docs directory (#2846)
IMAP-Science-Operations-Center/imap_processing: docs/PERMISSIONS.md

# Repository Access & Permissions Guide

> This document outlines the different permission levels available in this repository and what each role can do.

Reviewer: ```suggestion
This document outlines the different permission levels available in IMAP repository and what access each role can provide.
```

## Overview

> GitHub provides five repository roles. Below is a detailed breakdown of permissions for each role, including what free GitHub users can access and what needs to be actively managed.

Reviewer: ```suggestion
GitHub provides five repository roles. Below is a detailed breakdown of permissions for each role, including what access is provided for any GitHub user and what needs to be requested from IMAP SDC team.
```

---

## Role Permissions & Responsibilities

> ### 1. Read (Viewer)

Reviewer: ```suggestion
### 1. Read
```

> **Use Case**: Issue reporters, documentation readers, community members

Reviewer: ```suggestion
```

> #### What Free GitHub Users Can Already Do

Reviewer: remove work "Free" in all of the content
```suggestion
#### What GitHub Users Can Already Do
```

- ✅ View repository content
- ✅ Create issues
- ✅ Comment on issues
- ✅ View pull requests
- ✅ Comment on PRs

#### What You Need to Manage
- ❌ No restrictions needed (this is a safe, read-only role)

#### Full Permission List
| Action | Allowed |
|--------|---------|
| View repository content | ✅ |
| Create issues | ✅ |
| Comment on issues and PRs | ✅ |
| View pull requests | ✅ |
| Push code | ❌ |
| Create PRs | ❌ |
| Merge anything | ❌ |

> **When to Use**: Add people who only need to report issues and view code.

Reviewer: remove these "when to use" lines in all content
```suggestion
```

---

> ### 2. Triage (Issue Manager)

Reviewer: apply similar pattern to remaining content
```suggestion
### 2. Triage
```

**Use Case**: Issue triagers, community managers, QA team members

#### What Free GitHub Users Can Already Do
- ❌ None of these actions

#### What You Need to Manage
- ✅ **Grant** all triage permissions (labels, assignees, milestones, close/reopen)

> - ❌ No restrictions needed (this is a safe role)

Reviewer: remove this section as well from all content
